content/markdown/02-working-with-data/02-06-factors.rmd

```diff
@@ -0,0 +1,189 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Factors - Releveling & Recoding
+**Author:** Cole Brookson
+**Date:** 20 July 2022
+
+```{r, echo=FALSE}
+options(tidyverse.quiet = TRUE)
+```
+
+
+## Intro to Factors 
+
+Factors are an important but frustrating special data type in R. They can be thought of as "categorical" variables, in the sense that they can only take on a limited number of values. R uses a vector of integer values to store a factor, with a corresponding set of character values that are displayed while using the factor.
```

Comment:
Could we soften the edges on the first sentence? What about "Factors are an important special data type in R, though they can be frustrating." ?

I think we should also start the second paragraph with "Factors" rather than "They"